### PR TITLE
feat: create parse Data Service

### PR DESCRIPTION
**Pull request by Chlenix**

### CODE REVIEW

File: src/fdk_rdf_parser/fdk_rdf_parser.py [hunk 1]
```diff
@@ -18,6 +19,20 @@ def isTypeDataset(graph: Graph, topic: URIRef) -> bool:
     return False
 
 
+def parseDataServices(dataServiceRDF: str) -> Dict[str, DataService]:
+    dataServices: Dict[str, DataService] = {}
+
+    dataServicesGraph = Graph().parse(data=dataServiceRDF, format='turtle')
+
+    for dataServiceURI in dataServicesGraph.subjects(
+            predicate=RDF.type, object=dcatURI("DataService")
```
Comment (NilsOveTen) [inline src/fdk_rdf_parser/fdk_rdf_parser.py:28]: use CatalogRecord instead of DataService, then you'll have metadata available

Comment (NilsOveTen) [reply src/fdk_rdf_parser/fdk_rdf_parser.py:28]: and you then need to copy the 'isTypeDataset' functionality

Comment (Chlenix) [reply src/fdk_rdf_parser/fdk_rdf_parser.py:28]: Do you mean these metadata?

image

Comment (Chlenix) [reply src/fdk_rdf_parser/fdk_rdf_parser.py:28]: And should I duplicate them to every data service?

Comment (NilsOveTen) [reply src/fdk_rdf_parser/fdk_rdf_parser.py:28]: duplicate?
They all have their own individual CatalogRecord

Comment (NilsOveTen) [reply src/fdk_rdf_parser/fdk_rdf_parser.py:28]: you might be using an old version of dataservice-harvester, recently added that metadata for DataService is included when you get a Catalog

**Pull request merged**